lib/baseTest.py

```diff
@@ -132,21 +137,48 @@ def remove_test_file(self):
         if hasattr(self, "test_url_id"):
             self.target_helper.delete_target(self.test_url_id)
 
-    def load_configs(self):
+    def platform_dep_settings_handler(self, config_value):
```

Comment:
`foo_handler` is a noun.
In general, we use noun for class name, not method name.